cmd/config.go

```diff
@@ -259,6 +259,9 @@ type tsdbDiscoveryOpts struct {
 	discoveryMinBlockAge time.Duration
 
 	externalLabelMatchers matcherSlice
+
+	minTimeOffset time.Duration
```

Comment:
since these flags are consumed by converter we should put them into convertOpts i think!